workspaces/bulk-import/plugins/bulk-import-backend/src/catalog/types.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export type CatalogLocation = {
+  id?: string;
+  target: string;
+  source: SourceType;
+};
+
+export type SourceType = 'location' | 'config' | 'other';
```

Comment:
How about using reusing the types generated off of the OpenAPI spec? Otherwise, we might end up out-of-sync if the OpenAPI spec is modified.
https://github.com/dzemanov/rhdh-plugins/blob/bulk-import-add-source/workspaces/bulk-import/plugins/bulk-import-backend/src/generated/openapi.d.ts#L233

Reply:
🙈 Och gott, yeah.